pyproject.toml

```diff
@@ -1,3 +1,38 @@
+[tool.poetry]
+name = "mage-ai"
+version = "0.8.75-dev"
+description = "DevEx poetry configuration"
+authors = ["Mage"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
+
+[tool.poetry.group.dev.dependencies]
+mypy = "*"
+black = "*"
+flake8 = "*"
+pre-commit = "*"
+pytest = "*"
+pytest-cov = "*"
+interrogate = "*"
+isort = "*"
+commitizen = "*"
+ipykernel = "^6.17.1"
+pandas-stubs = "*"
+flake8-bugbear = "*"
+sourcery = "*"
+
+[tool.isort]
+profile = "black"
+import_heading_stdlib = "Python Standard Library Imports"
```

Comment:
we don't need the heading.

Reply:
Okay, I'll remove the `isort` headings for imports.